gradle/libs.versions.toml

```diff
@@ -0,0 +1,142 @@
+ktor = "2.0.2"
+moshi = "1.12.0"
+moshix = "0.14.1"
+okHttp = "4.9.3"
+rx-android = "2.0.1"
+rx-java2 = "2.2.21"
+rx-java3 = "3.1.3"
+sqldelight = "1.5.3"
+truth = "1.1.3"
+
+[libraries]
+android-plugin = { group = "com.android.tools.build", name = "gradle", version.ref = "android-plugin" }
+android-plugin-min = { group = "com.android.tools.build", name = "gradle", version.ref = "android-plugin-min" }
+android-support-annotations = { group = "com.android.support", name = "support-annotations", version = "28.0.0" }
+android-test-runner = { group = "com.android.support.test", name = "runner", version = "1.0.2" }
+androidx-appcompat = { group = "androidx.appcompat", name = "appcompat", version = "1.1.0" }
+androidx-espresso-idlingResource = { group = "androidx.test.espresso", name = "espresso-idling-resource", version = "3.4.0" }
+androidx-recyclerView = { group = "androidx.recyclerview", name = "recyclerview", version = "1.1.0" }
+androidx-sqlite = { group = "androidx.sqlite", name = "sqlite", version.ref = "androidx-sqlite" }
+androidx-sqlite-framework = { group = "androidx.sqlite", name = "sqlite-framework", version.ref = "androidx-sqlite" }
+androidx-startup-runtime = { group = "androidx.startup", name = "startup-runtime", version = "1.1.1" }
+antlr = { group = "org.antlr", name = "antlr4", version.ref = "antlr" }
+antlr-runtime = { group = "org.antlr", name = "antlr4-runtime", version.ref = "antlr" }
+apollo-api = { group = "com.apollographql.apollo3", name = "apollo-api", version.ref = "apollo" }
+apollo-adapters = { group = "com.apollographql.apollo3", name = "apollo-adapters", version.ref = "apollo" }
+apollo-httpCache = { group = "com.apollographql.apollo3", name = "apollo-http-cache", version.ref = "apollo" }
+apollo-idlingResource = { group = "com.apollographql.apollo3", name = "apollo-idling-resource", version.ref = "apollo" }
+apollo-plugin = { group = "com.apollographql.apollo3", name = "apollo-gradle-plugin", version.ref = "apollo" }
+apollo-runtime = { group = "com.apollographql.apollo3", name = "apollo-runtime", version.ref = "apollo" }
+apollo-rx2 = { group = "com.apollographql.apollo3", name = "apollo-rx2-support", version.ref = "apollo" }
+apollo-mockserver = { group = "com.apollographql.apollo3", name = "apollo-mockserver", version.ref = "apollo" }
+apollo-mppUtils = { group = "com.apollographql.apollo3", name = "apollo-mpp-utils", version.ref = "apollo" }
+apollo-normalizedCache = { group = "com.apollographql.apollo3", name = "apollo-normalized-cache", version.ref = "apollo" }
+apollo-normalizedCache-incubating = { group = "com.apollographql.apollo3", name = "apollo-normalized-cache-incubating", version.ref = "apollo" }
+apollo-normalizedCache-sqlite = { group = "com.apollographql.apollo3", name = "apollo-normalized-cache-sqlite", version.ref = "apollo" }
+apollo-normalizedCache-sqlite-incubating = { group = "com.apollographql.apollo3", name = "apollo-normalized-cache-sqlite-incubating", version.ref = "apollo" }
```

Comment:
Nit: What do you say about we stick to kebab-case every where instead of a mix here? Doesn't have to be done now. but as we move forward?

Reply:
Yup, not a big deal, but my reasoning was that the `-` are used to break up the generated fields and so it makes sense for them to be logically grouped, and for instance `apollo.normalized.cache` doesn't make much sense because `normalized` is not a thing. But we can keep `normalizedcache` in lowercase though!